src/quo2/components/dividers/date.cljs

```diff
@@ -0,0 +1,17 @@
+(ns quo2.components.dividers.date
+  (:require [react-native.core :as rn]
+            [clojure.string :as string]
+            [quo2.components.separator :as separator]
+            [quo2.components.markdown.text :as text]
+            [quo2.foundations.colors :as colors]))
+
+(defn date [value]
+  [rn/view {:margin-vertical 16
+            :padding-left 52}
+   [text/text {:weight              :medium
+               :accessibility-label :message-datemark-text
```

Comment:
About the accessibility label:

The divider date component is very general and shouldn't know about *messages*, so I think this should be reflected in this accessibility label. It's only used for testing at the moment, but something like `:divider-date-text` or `:divider-date-value` seems more appropriate.